src/LinuxTracing/TracerImpl.cpp

```diff
@@ -677,6 +679,20 @@ bool TracerImpl::OpenInstrumentedTracepoints(const std::vector<int32_t>& cpus) {
   return !tracepoint_event_open_errors;
 }
 
+// We maintain a map of pids from the target process pid namespace to the root namespace. When a new
```

Comment:
You mention pids in this comment but tids everywhere else. Shouldn't it be tids?